README.md

```diff
@@ -120,17 +120,17 @@ After configuring the build system:
 
 ## Using
 ### Querying
-The Search service can be queried for Products. See [Query Service API](https://github.com/connexta/ion-query-api) for information about paths and parameters
+The Search service can be queried for IRM. See [Query Service API](https://github.com/connexta/ion-query-api) for information about paths and parameters
 The Search service supports the OGC Catalogue Common Query Language (OGC CommonQL).
 See [Annex B - BNF Definition of OGC CommonQL](http://docs.opengeospatial.org/is/12-168r6/12-168r6.html#62) for the definition of the grammar.
 
 The following are valid query attributes:
 
 | Attribute | Description  |
 |---|---|
-| `id` | The unique ID that is associated with each Product |
+| `id` | The unique ID that is associated with each IRM |
 | `contents` | A client can perform keyword queries with this attribute. |
-| `media_type` | The media (MIME) type of the Product |
+| `media_type` | The media (MIME) type of the IRM |
```

Comment:
❗️`file`

Reply:
Actually, this _is_ currently the mediatype of the file attachment, which is the IRM. I will change this back. 🤦‍♀